start/sw/web-app/laravel5.md

```diff
@@ -57,7 +64,7 @@ $ laravel new ProjectName
 ```bash
 $ cd ProjectName
 $ cp .env.example .env
-$ nano .env
+$ gedit .env #or nano,vim,emacs
```

Comment:
Please move the comment to line above.
I'd say that there's no need to list the editors, but you can describe what you're about to do, e.g. `edit file XYZ`.

_Note: Using one or other is just a preference, and AFAIK `gedit` is in default installation. So it's fine with me._